src/LHScriptX/Script.cpp

```diff
@@ -119,7 +120,7 @@ ScriptCommandParameter GetParameter(Token& argument)
 				const auto z = std::strtof(floatEnd + 1, &floatEnd);
 				if (static_cast<size_t>(floatEnd - str.c_str()) == static_cast<size_t>(str.length()))
 				{
-					const auto& island = Game::Instance()->GetLandIsland();
+					const auto& island = Locator::terrainSystem::value();
```

Comment:
Now if script hasn't yet parsed `LOAD_LANDSCAPE`, this results in an assert fail.

Reply:
fixed on the next commit